src/injection/react-native-web.js

```diff
@@ -10,7 +10,7 @@ const {
   Dimensions,
   Easing,
 } = require('react-native-web');
-const StyleRegistry = require('react-native-web/dist/apis/StyleSheet/registry');
+const StyleRegistry = require('react-native-web/dist/apis/StyleSheet/registry').default;
```

Comment:
Yes, that's true :-) using import instead of require is a good option here; adding add-module-exports would fix it if it's only got a default export.